addons/mail/static/src/models/activity_view/activity_view.js

```diff
@@ -3,6 +3,11 @@
 import { registerModel } from '@mail/model/model_core';
 import { attr, many2one, one2one } from '@mail/model/model_field';
 import { clear, insertAndReplace } from '@mail/model/model_field_command';
+import {
+    auto_str_to_date,
+    getLangDateFormat,
+    getLangDatetimeFormat,
+} from 'web.time';
```

Comment:
should have new line above imports from different modules (also I would have this one in a single line)